api/src/room_store.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+
+
+class RoomStore:
+    def __init__(self):
+        self.path = os.environ['ROOM_STORE_DIR']
```

Comment:
This should be a constructor argument. The only thing that should read environment variables is your entrypoint

Reply:
Okay, fine